tests/bearlib/languages/LanguageTest.py

```diff
@@ -27,3 +27,9 @@ def test_invalid_attribute(self):
     def test_attribute_list_empy(self):
         with self.assertRaisesRegex(AttributeError, 'no available attribute'):
             self.lang_unknown.not_an_attribute
+
+    def test__reduce__(self):
+        kallable, args = self.lang_cpp.__reduce__()
```

Comment:
This is testing *your* code, but it is not testing pickle-ability.
You want to use `pickle` to testing it can be pickled.
see https://github.com/wikimedia/pywikibot-core/blob/master/tests/page_tests.py#L512